### ISSUE TITLE
How to further improve the effect of tse？

### ISSUE THREAD

**Issue by KollyYang** [mcw519/PureSound#3]
Hi 
Thank you very much for open-sourcing this project. This is a great open-source project, which is very helpful to me, because I just came into contact with tse recently, but currently there are relatively few tse projects/demo  on github.

I first carefully read the source code, and then ran tse/demo_app, using skim_causal_460_wNoise_IS_tsdr.ckpt to do the test, the effect is not very satisfactory.

At the same time, the above model was also used for offline testing. Using the synth data in https://github.com/eeskimez/pse-samples for comparison, it was found that the effect was worse than that of pdcattunet.

Based on your current tse results, is there any possibility to improve the effect? Do you have some suggestions?
Thanks!

**Comment by mcw519** [comment]
Hi,
Can you share the testing data and its related enrollment speech to me?
Let me check the results first.
Thanks.

**Comment by KollyYang** [comment]
Hi 
Are these synthesized [audios](https://github.com/mcw519/PureSound/files/11684516/synth.zip)okay? 
Thanks!

**Comment by mcw519** [comment]
Hi Kolly,

The synthesized data is okay but looks there are no enrollment speech can enroll the speaker for TSE task.
I thinks the folder including:
  xxx_no_processing.wav -> input audio
  xxx_ref.wav -> ground truth
and the others are different model's output results.

Thanks.

**Comment by KollyYang** [comment]
Hi Wu
Thanks a lot, Can xxx_ref.wav be used as enrollment speech?

**Comment by mcw519** [comment]
Thanks you provided the real cases.
I ran the inference through the evaluation mode in main.py and get the results here [proc.zip](https://github.com/mcw519/PureSound/files/11699431/proc.zip).
I think the pre-trained model does not has good robustness to handle the music noises because the noise corpus is WHAM.

**Comment by KollyYang** [comment]
Hi Wu
Thanks, this is results [voice-filter_proc.zip](https://github.com/mcw519/PureSound/files/11702060/voice-filter_proc.zip)  results ..
